tests/Unit/InlineSelectLensTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithFaker;
+use KirschbaumDevelopment\Nova\InlineSelect;
+
+class InlineSelectLensTest extends TestCase
+{
+    use WithFaker;
+
+    /**
+     * @var InlineSelect
+     */
+    protected InlineSelect $inlineSelect;
+
+    /**
+     * @before
+     */
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->inlineSelect = new InlineSelect('inline-select');
+    }
+
+    public function testThatItInlinesOnLens()
+    {
+        $this->inlineSelect->inlineOnIndex();
```

Comment:
@brandonferens Does it makes sense to call the `inlineOnLens ` even if it calls this method eventually?